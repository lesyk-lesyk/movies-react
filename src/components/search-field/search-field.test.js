import React from 'react';
import { SearchField } from './index';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store'

describe('SearchField', () => {
  it('renders', () => {
    const store = configureStore()({ searchForm: { term: 'searchTerm' } });
    const tree = renderer
      .create(<SearchField store={store} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
