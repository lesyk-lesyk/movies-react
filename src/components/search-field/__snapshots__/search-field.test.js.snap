// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField renders 1`] = `
<form
  className="search-form"
  onSubmit={[Function]}
>
  <input
    className="search-field"
    onChange={[Function]}
    placeholder="Enter search term..."
    type="search"
    value="searchTerm"
  />
  <span
    className="search-field-icon"
  >
    â†©
  </span>
</form>
`;
